lib/temporal/workflow/context.rb

```diff
@@ -16,7 +17,7 @@ module Temporal
   class Workflow
     class Context
       attr_reader :metadata, :config
-
+      
```

Comment:
Nit: I think your editor is not stripping trailing whitespaces, leaving these artifacts behind

Reply:
Ah, thanks, been meaning to fix that in this workspace.